packagingbuild/Dockerfile.template

```diff
@@ -136,13 +136,13 @@ RUN PATH=/usr/share/python/st2python/bin:$PATH bash -c \
 {%- elif dist in ('centos', 'fedora') -%}
 
 # Install development tools and python for EL8
-RUN yum -y install python{%- if (version == '8') -%}3 python2 python3{%- endif %}-devel rpmdevtools && \
+RUN yum -y install python{%- if (version == '8') -%}3 python2 python2-devel gcc python3 {%- endif%}-devel rpmdevtools && \
```

Comment:
Looks like this template would generate `python3 -devel`

Reply:
```suggestion
RUN yum -y install python{%- if (version == '8') -%}3 python2 python2-devel gcc python3{%- endif %}-devel rpmdevtools && \
```

Reply:
It actually didn't matter, but I'm pushing an update to address it.